packages/subgraph/src/utils/constants.ts

```diff
@@ -27,6 +27,6 @@ export const TOKEN_VOTING_INTERFACE = '0x50eb001e';
 export const ADDRESSLIST_VOTING_INTERFACE = '0x5f21eb8b';
 export const ADMIN_INTERFACE = '0xa5793356';
 export const MULTISIG_INTERFACE = '0x8f852786';
-export const WRAPPED_ERC20_INTERFACE = '0x12cdd3ac';
+export const WRAPPED_ERC20_INTERFACE = '0x0f13099a';
```

Comment:
If the interface ID had changed, wouldn't it make sense to keep supporting the old one as well? Existing tokens might be using it as well.

Reply:
Make sense, however APP reported they never created it, since it was not supported,
and we are not sure if this interface have changed or just put there wrongly.